client_coverage_test.go

```
@@ -0,0 +1,348 @@
+}
+
+func TestRetryBackoffBounds(t *testing.T) {
+	t.Run("disabled", func(t *testing.T) {
+		if got := retryBackoff(4, 0, time.Second); got != 0 {
+			t.Fatalf("retryBackoff() = %s, want 0", got)
+		}
+	})
+
+	t.Run("jittered", func(t *testing.T) {
+		const (
+			min = 10 * time.Millisecond
```

golangci-lint (2.12.2)

[error] const min has same name as predeclared identifier

(predeclared)